Frameworks/Foundation/NSURLProtocol_WinHTTP.mm

```diff
@@ -251,7 +251,7 @@ - (void)startLoading {
 
         NSError* error = nil;
         try {
-            auto httpRequestMessage = _requestMessageForNS(_request);
+            auto httpRequestMessage = _requestMessageForNS(self.request);
```

Comment:
NSURLProtocol_WinHTTP doesn't have access to the superclass member now that it's synthesized